platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/AbstractGossip.java

```diff
@@ -141,8 +141,7 @@ protected AbstractGossip(
         final CryptoConfig cryptoConfig = platformContext.getConfiguration().getConfigData(CryptoConfig.class);
         final SocketConfig socketConfig = platformContext.getConfiguration().getConfigData(SocketConfig.class);
 
-        topology = new StaticTopology(
-                addressBook, selfId, basicConfig.numConnections(), unidirectionalConnectionsEnabled());
+        topology = new StaticTopology(addressBook, selfId, basicConfig.numConnections());
```

Comment:
I don't see where the topology is changed later in the code; if I didn't miss it, maybe declare as  final ?

Reply:
topology is already final. This is old code, I just updated the params to reflect the new constructor signature.